.github/workflows/pylint.yml

```
@@ -21,11 +23,20 @@ jobs:
         python get-pip.py
         pip install pylint==1.1.0
         pip install astroid==1.2
-        
+
     - name: Run Pylint Checks
       run: |
-        pylint --rcfile=.pylintrc convert2rhel/
-        
+         function interpreted_pylint() {
+            local es=0
+            pylint --rcfile=.pylintrc convert2rhel/; es=$?
+            errors=$(( es&3 ))
+            case $errors in
+                 1|2|3)  return $es ;;
+            *) return 0 ;;
+            esac
+         }
+         interpreted_pylint
```

@fellipeh Could we try something like this?
```suggestion
         function interpreted_pylint() {
            local es=0
            pylint --rcfile=.pylintrc convert2rhel/; es=$?
            if ! [ $((es & 3)) -eq 0 ]; then
                return $es;
            fi;
            return 0;
         }
         interpreted_pylint
```

---

@fellipeh please, take a look previous comment from @SpyTec ;)